server.js

```diff
@@ -36,7 +36,7 @@ app.use(
       tableName: "session",
     }),
     secret: process.env.SESSION_SECRET,
-    resave: false,
+    resave: true, //check if you want this
```

Comment:
i dont know if you want this to be true, it was false before

Reply:
I did want it to be false. Why did you want it to be true?